core-bundle/contao/languages/en/default.xlf

```diff
@@ -1520,6 +1520,9 @@
       <trans-unit id="MSC.copySelected">
         <source>Copy</source>
       </trans-unit>
+      <trans-unit id="MSC.copyMultiple">
+        <source>Copy multiple</source>
```

Comment:
The label is misleading IMHO, because `Copy` also copies multiple elements. The new action copies multiple elements multiple times, so the label should say _copy and keep_ or _copy multiple times_ or something alike.